src/components/tooltip/tooltip.tsx

```diff
@@ -41,8 +43,10 @@ import { createRandomString } from '../../util/random-string';
  * effortlessly recognize can be hovered.
  *
  * @exampleComponent limel-example-tooltip
+ * @exampleComponent limel-example-tooltip-max-character
  * @private
  */
+
 @Component({
     tag: 'limel-tooltip',
```

Comment:
I wonder if you have some incorrect setting in your editor, because I've seen a blank line added between the docblock and the `@Component` decorator, just like this, on loads of your PRs. Or is it just a habit that's hard to kill? 😄

Reply:
this might have come from my bad habits 😄

Reply:
yeah, lint fix didn't correct this